### ISSUE TITLE
GIS Support Street Class - L. Altamirano

### ISSUE THREAD

**Issue by jaime-mckeown** [cityofaustin/atd-geospatial#136]
Received the following email from Leslie:

> According to Public Works Department and the Office of Real Estate Services Stratford Drive through Zilker Park (see below) is a park road or a “private drive”; however on GIS it is shown as a City Collector in its entirety so I am requesting for this to be corrected and updated. If you are not the correct contact please redirect me at your discretion.

**Comment by jaime-mckeown** [comment]
I looked up the street in the street segment layer to double check, then asked Diana about process with CTM getting it changed. She said that the user should submit the ticket as it is effecting them. I let Leslie know that she needs to submit a CTM ticket to get that changed in the street segment layer.

I also looked at the new atd maintained streets layer that Signs & Markings just added to GISDM and it shows the maintenance agency. This would be a good layer to add to the ATD GIS Reference Web Map. I will need to check with them about adding it to AGOL.
